multihost_test/bz_automation/test_automation.py

```diff
@@ -185,3 +185,55 @@ def test_pam_faillock_audit(self, multihost, create_localusers, bkp_pam_config):
         log_str = multihost.client[0].get_file_contents("/var/log/audit/audit.log").decode('utf-8')
         client.run_command("cp -vf /etc/security/faillock.conf_anuj /etc/security/faillock.conf")
         assert f'op=pam_faillock suid={uid}' in log_str
+
+    def test_6967(self, multihost, bkp_pam_config):
+        """
+        :title: Changing UID_MIN ignores GID_MIN value in /etc/login.defs
+        :id: c6f49b14-9262-11ee-a30b-845cf3eff344
+        :bugzilla: https://issues.redhat.com/browse/SSSD-4535
```

Comment:
Change ticket to https://bugzilla.redhat.com/show_bug.cgi?id=2070532, and also change it in the commit message

Reply:
You didn't change the commit message so I'm reopening this one.